ponyup-init.sh

```diff
@@ -3,7 +3,12 @@
 set -o errexit
 set -o nounset
 
-default_prefix="$HOME/.local/share"
+if [ -z "$XDG_DATA_HOME" ]; then
+    default_prefix="$HOME/.local/share"
```

Comment:
The indentation is off here. The rest of the file uses 2 spaces.

Reply:
Fixed, forgot to put it here though (whoops).